Makefile

```diff
@@ -28,6 +28,8 @@ N_THREADS ?= $(shell nproc)
 
 #### Setup ####
 
+export LANG := C
```

Comment:
Could you add a comment explaining why this is needed?

Reply:
done :)